diff --git a/node_modules/@lekoarts/gatsby-theme-emilia/src/components/footer.tsx b/node_modules/@lekoarts/gatsby-theme-emilia/src/components/footer.tsx
index 0a6c078..4463aed 100755
--- a/node_modules/@lekoarts/gatsby-theme-emilia/src/components/footer.tsx
+++ b/node_modules/@lekoarts/gatsby-theme-emilia/src/components/footer.tsx
@@ -2,18 +2,13 @@
 import { Box, jsx, Container, Flex, Link, useColorMode } from "theme-ui"
 import useEmiliaConfig from "../hooks/use-emilia-config"
 import SocialMediaList from "./social-media-list"
-import ColorModeToggle from "./colormode-toggle"
 // @ts-ignore
 import AboutMeMDX from "../texts/about-me"
 
 const Footer = () => {
   const { showThemeAuthor } = useEmiliaConfig()
-  const [colorMode, setColorMode] = useColorMode()
+  const [colorMode] = useColorMode()
   const isDark = colorMode === `dark`
-  const toggleColorMode = (e: any) => {
-    e.preventDefault()
-    setColorMode(isDark ? `light` : `dark`)
-  }
 
   return (
     <Box
@@ -24,7 +19,7 @@ const Footer = () => {
       }}
     >
       <Container>
-        <div sx={{ display: `grid`, gridGap: 4, gridTemplateColumns: [`1fr`, `1fr`, `1fr`, `2fr 1fr`] }}>
+        <div sx={{ display: `grid` }}>
           <div
             sx={{
               p: { mb: 0 },
@@ -36,14 +31,18 @@ const Footer = () => {
           >
             <AboutMeMDX />
           </div>
+        </div>
+      </Container>
+
+      <Container>
+        <div>
           <Flex
             sx={{
-              textAlign: [`center`, `center`, `center`, `right`],
+              textAlign: `center`,
               flexDirection: `column`,
               justifyContent: `space-between`,
             }}
           >
-            <ColorModeToggle isDark={isDark} toggle={toggleColorMode} />
             <div sx={{ mt: [4, 4, 4, 0] }}>
               <div sx={{ a: { ml: [1, 1, 1, 2], mr: [1, 1, 1, 0] } }}>
                 <SocialMediaList />
@@ -53,6 +52,7 @@ const Footer = () => {
           </Flex>
         </div>
       </Container>
+
       {showThemeAuthor && (
         <Container
           sx={{
diff --git a/node_modules/@lekoarts/gatsby-theme-emilia/src/components/header.tsx b/node_modules/@lekoarts/gatsby-theme-emilia/src/components/header.tsx
index 816c79a..a4f4bdf 100755
--- a/node_modules/@lekoarts/gatsby-theme-emilia/src/components/header.tsx
+++ b/node_modules/@lekoarts/gatsby-theme-emilia/src/components/header.tsx
@@ -1,5 +1,5 @@
 /** @jsx jsx */
-import { jsx, Heading, Flex } from "theme-ui"
+import { jsx, Heading, Flex, useColorMode } from "theme-ui"
 import { animated, useSpring, config } from "react-spring"
 import { useStaticQuery, graphql } from "gatsby"
 import { GatsbyImage, IGatsbyImageData } from "gatsby-plugin-image"
@@ -8,6 +8,8 @@ import HeaderBackground from "./header-background"
 import Location from "../assets/location"
 import SocialMediaList from "./social-media-list"
 
+import ColorModeToggle from "./colormode-toggle"
+
 type AvatarStaticQuery = {
   file: {
     childImageSharp: {
@@ -17,6 +19,15 @@ type AvatarStaticQuery = {
 }
 
 const Header = () => {
+
+  const [colorMode, setColorMode] = useColorMode()
+  const isDark = colorMode === `dark`
+  
+  const toggleColorMode = (e: any) => {
+    e.preventDefault()
+    setColorMode(isDark ? `light` : `dark`)
+  }
+
   const { name, location, assetsPath } = useEmiliaConfig()
   const avatar = useStaticQuery<AvatarStaticQuery>(graphql`
     query {
@@ -105,6 +116,19 @@ const Header = () => {
             <Location /> {location}
           </Flex>
         </animated.div>
+        <animated.div style={fadeUpPropsDelay}>
+          <Flex
+            sx={{
+              mt: 3,
+              mb: 2,
+              justifyContent: `center`,
+              alignItems: `center`,
+              color: `text`,
+            }}
+          >
+            <ColorModeToggle isDark={isDark} toggle={toggleColorMode} />
+          </Flex>
+        </animated.div>
         <div data-testid="social-header" sx={{ mt: 4, mb: 6, a: { mx: 2 } }}>
           <animated.div style={fadeLongProps}>
             <SocialMediaList />
