// @lekoarts/gatsby-theme-emilia-core/gatsby-config.js
const withDefaults = require(`./utils/default-options`)

module.exports = (themeOptions) => {
  const options = withDefaults(themeOptions)

  return {
    siteMetadata: {
      siteTitle: `Naoto Nishida`,
      siteTitleAlt: `Naoto Nishida | 西田 直人`,
      siteHeadline: `Naoto Nishida's portfolio site`,
      siteUrl: `https://nawta.github.io`,
      siteDescription: `ポートフォリオのサイトです`,
      siteLanguage: `en`,
      siteImage: `/wanko_banner.jpg`,
      author: `Naoto Nishida`,
      basePath: options.basePath,
      projectsPath: options.projectsPath,
    },
    plugins: [
      {
        resolve: `gatsby-source-filesystem`,
        options: {
          name: options.projectsPath,
          path: options.projectsPath,
        },
      },
      {
        resolve: `gatsby-source-filesystem`,
        options: {
          name: options.assetsPath,
          path: options.assetsPath,
        },
      },
      options.mdx && {
        resolve: `gatsby-plugin-mdx`,
        options: {
          lessBabel: true,
        },
      },
      `gatsby-transformer-sharp`,
      options.sharp && {
        resolve: `gatsby-plugin-sharp`,
        options: {},
      },
    ].filter(Boolean),
  }
}
